<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QLHoatDong</class>
 <widget class="QWidget" name="QLHoatDong">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qu·∫£n l√Ω Ho·∫°t ƒë·ªông ƒêo√†n</string>
  </property>
  <widget class="QPushButton" name="btnXoa">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>üóë X√≥a</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSua">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>‚úèÔ∏è S·ª≠a</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnThem">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>‚ûï Th√™m</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableDoanVien">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>801</width>
     <height>231</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <column>
    <property name="text">
     <string>M√£ ƒëo√†n vi√™n</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>H·ªç t√™n</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Chi ƒëo√†n</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Khoa</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Kh√≥a h·ªçc</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableHoatDong">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>801</width>
     <height>231</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>T√™n ho·∫°t ƒë·ªông</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ng√†y t·ªï ch·ª©c</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ƒê·ªãa ƒëi·ªÉm</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>N·ªôi dung</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnTimKiem">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>üîç T√¨m ki·∫øm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTimKiem">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>137</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="placeholderText">
    <string>T√¨m ki·∫øm ho·∫°t ƒë·ªông...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Qu·∫£n l√Ω ho·∫°t ƒë·ªông</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>345</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ƒêo√†n vi√™n tham gia:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnXoaDV">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>630</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>üóë X√≥a</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnThemDV">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>630</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>‚ûï Th√™m</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtThemDV">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>630</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Nh·∫≠p m√£ ho·∫∑c t√™n ƒëo√†n vi√™n</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
